torch/jit/frontend.py

```diff
@@ -418,18 +418,17 @@ class ExprBuilder(Builder):
 
     @staticmethod
     def build_Attribute(ctx, expr):
-        # NB: the only attributes we support are for getting methods
-        value = build_expr(ctx, expr.value)
-        # <sigh> name is just a string, so it's not annotated in any way.
-        source = ctx.source
-        pos = find_after(ctx, value.range().end, '.').end  # Start with the dot
-        while source[pos] in string.whitespace:  # Skip whitespace
-            pos += 1
-        start_pos = pos
-        while source[pos] in _identifier_chars:  # Find the identifier itself
-            pos += 1
-        name_range = ctx.make_raw_range(start_pos, pos)
-        return Select(value, Ident(name_range, expr.attr))
+        base = build_expr(ctx, expr.value)
```

Comment:
Why is the code in `build_Attribute` triggering this? What does it have to do with comments?

Reply:
Pybind auto converts strings to utf-8, so the index returned from the source range is an index by utf-8 bytes. Emojis for example turn into 4 bytes, but when indexing in Python in a `str` they’re treated as 1 character. So the indices used here are mismatched hence the error in the issue